src/utils/getAccuracyData.ts

```diff
@@ -0,0 +1,21 @@
+import { TAccuracyStatistics } from '@/contexts/AccuracyContext.types'
+import { currentConfig } from './appconstants'
+import { accuracyStatisticsEndpoint } from './endpoints/accuracyEndpoints'
+import { Maybe } from './utils'
+
+export const getAccuracyData = async (): Promise<Maybe<TAccuracyStatistics>> =>
+  fetch(accuracyStatisticsEndpoint(currentConfig.websocketURL))
+    .then((response) => response.json())
+    .then((response) => {
+      // Response can return an error
+      const errorMessage = response?.message
+
+      // check if "No data available" is inside message so it can be handled correctly
+      if (errorMessage && errorMessage.includes('No data available'))
+        return null
+
+      return response
+    })
+    .catch((error) => {
+      console.error(error)
+    })
```

Comment:
Can we rename this file to something more generic, so it won't only contain just 1 function? Maybe name it 'getPdrBackendData'

Reply:
1. I agree that this could be in `/pdr-backend/getAccuracyData.ts`
1. I don't know if we should have 1 file per function.

Reply:
I am going to fix it

Reply:
I have changed the filename, if we have different endpoints and getters in future, we can consider how can we separate them